R/profvis.R

```diff
@@ -109,6 +117,16 @@ profvis <- function(expr = NULL,
     message("Intervals smaller than ~5ms will probably not result in accurate timings.")
   }
 
+  if (is.null(timing)) {
+    if (getRversion() >= "4.4.0" || Sys.info()$sysname == "Windows") {
```

Comment:
To fix the `$ operator is invalid for atomic vectors` error.

```suggestion
    if (getRversion() >= "4.4.0" || Sys.info()[["sysname"]] == "Windows") {
```